contracts/TradetrustERC721.sol

```diff
@@ -47,4 +70,8 @@ contract calculateTradeTrustERC721Selector {
       i.destroyToken.selector ^
       i.sendToken.selector;
   }
-}
\ No newline at end of file
+}
+
+
+
+
```

Comment:
Is lint not catching this extra lines?

Reply:
done. apparently not, i ran 'npm run lint' on vs code also didnt show. but i removed it alr.